src/pybind/mgr/cephadm/module.py

```diff
@@ -205,6 +205,15 @@ class CephadmOrchestrator(orchestrator.Orchestrator, MgrModule,
                          'can allow debugging daemons that encounter problems '
                          'at runtime.',
         },
+        {
+            'name': 'container_init',
+            'type': 'bool',
+            'default': False,
+            'desc': 'Run podman/docker with `--init`',
+            'long_desc': 'Kernel treats any process with PID 1 different. Especially '
```

Comment:
Could you please change the description? It sounds weird and does not explain the actual problem. The actual problem is that if you want your application function well inside the container environment you need an init process. You need an init process because a) reap zombies b) generate core dumps correctly. If any process with PID 1 is not able to handle signals as init does, then application (any ceph-* in this case) should not be started as init (i.e. with PID 1). So obviously this should be the default behavior, here I fully agree with Nathan.

Reply:
removed `long_desc`. We've talked about this topic in the orchestrator weekly. The idea is to make this mandatory, not just the default. 

But I can't do that in this PR. Thus keeping the flag here for now.